src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterpriseStart.scala

```diff
@@ -16,12 +16,14 @@
 
 package io.gatling.sbt.settings.gatling
 
+import java.util.{ Map => JMap }
```

Comment:
Our current convention is to remap the package instead and then use it as a prefix:

import java.{ util => ju }

ju.Map